libsql-server/src/rpc/proxy.rs

```diff
@@ -633,12 +633,6 @@ impl Proxy for ProxyService {
                     tracing::debug!("connected: {client_id}");
                     match connection_maker.create().await {
                         Ok(conn) => {
-                            if !conn.is_primary() {
```

Comment:
I removed this since this one actually didn't make much sense, we allow this in the stream_exec. I added an error message for the describe one which should catch schema changes.

Reply:
Okay this allows embedded replica's to "write" proxied to the primary for schema changes but does not allow them to sync.